introduction.md

```diff
@@ -2,4 +2,8 @@ Hi it's Dave:
 ~~~~
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?